File: src/Services/StatementProcessingService/StatementProcessingService.Application/Dtos/Response/Entries/BankStatementEntryResponse.cs
using System;

namespace StatementProcessingService.Application.Dtos.Response.Entries;

public class BankStatementEntryResponse
{
    public Guid Id { get; set; }
    public string AccountNumber { get; set; }
    public decimal InitialBalanceActive { get; set; }
    public decimal InitialBalancePassive { get; set; }
    public decimal TurnoverDebit { get; set; }
    public decimal TurnoverCredit { get; set; }
    public decimal FinalBalanceActive { get; set; }
    public decimal FinalBalancePassive { get; set; }
}

File: src/Client/test-task/src/app/models/responses/bank-statement-entry.response.ts
export interface BankStatementEntryResponse {
  id: string;
  accountNumber: string;
  initialBalanceActive: number;
  initialBalancePassive: number;
  turnoverDebit: number;
  turnoverCredit: number;
  finalBalanceActive: number;
  finalBalancePassive: number;
}
